server/base/src/main/java/org/apache/accumulo/server/fs/VolumeManager.java

```diff
@@ -135,6 +135,18 @@ FSDataOutputStream createSyncable(Path logPath, int buffersize, short replicatio
   // forward to the appropriate FileSystem object
   boolean mkdirs(Path path, FsPermission permission) throws IOException;
 
+  // forward to the appropriate FileSystem object
+  boolean mkdirs(Path path, String storagePolicy, String encoding) throws IOException;
```

Comment:
I saw a Policies object in another part of the code.  Could pass that instead of the two String parameters, making this internal API more narrowly typed.

Reply:
I think the 2 strings worked better with an earlier version of the Tablet code.  I'll have to move the Policies class from the master source to base, but should be doable.